src/mapblock_mesh.cpp

```diff
@@ -1215,7 +1216,7 @@ MapBlockMesh::MapBlockMesh(MeshMakeData *data, v3s16 camera_offset):
 		// Create meshbuffer
 		// This is a "Standard MeshBuffer",
 		// it's a typedeffed CMeshBuffer<video::S3DVertex>
-		scene::SMeshBuffer *buf = new scene::SMeshBuffer();
+		scene::SMeshBufferTangents *buf = new scene::SMeshBufferTangents();
```

Comment:
It slows down rendering because it's more memory to be allocated for something that's only used if a user has a very specific setting enabled (95% do not).  I hope you understand why I'd be hesitant on making this the default.